crates/linter/src/runner.rs

```diff
@@ -77,7 +86,18 @@ impl<'a> Runner<'a> {
             issues: &mut self.issues,
         };
 
-        lint_ast_node(&configured_rule.rule, &self.ast, &mut context);
+        let mut ignores = self
+            .ignores
+            .iter()
+            .filter(|ignore_comment| {
+                ignore_comment
+                    .directives
+                    .iter()
+                    .any(|directive| configured_rule.slug.eq_ignore_ascii_case(directive.rule))
+            })
+            .collect::<Vec<_>>();
+
+        lint_ast_node(configured_rule, &self.ast, &mut context, &mut ignores);
```

Comment:
TODO: any `Ignore` left in the vec after this point must be reported as an issue ( useless ignore comment ).